### PR TITLE
Chapter 6 : Modelling User 

### PR DESCRIPTION
**Pull request by mohdzakwan99**
ready

### CODE REVIEW

File: test/models/user_test.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```
Comment (anlvq-1058) [inline test/models/user_test.rb:1]: delete test file

File: config/database.yml [hunk 2]
```diff
@@ -1,25 +1,26 @@
...
-  <<: *default
-  database: db/development.sqlite3
-
-# Warning: The database defined as "test" will be erased and
-# re-generated from your development database when you run "rake".
-# Do not set this db to the same as development or production.
 test:
-  <<: *default
-  database: db/test.sqlite3
+  adapter: mysql2
```
Comment (anlvq-1058) [inline config/database.yml:2]: use git ignore to ignore this file please

**Review by anlvq-1058 — COMMENTED** [review]
you have 2 commit please combine to 1

**Comment by mohdzakwan99** [comment]
ready

**Comment by mohdzakwan99** [comment]
ready

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -1,7 +1,11 @@
 module ApplicationHelper
+<<<<<<< HEAD
```
Comment (ngocvt-0484) [inline app/helpers/application_helper.rb:2]: please fix conflict

**Pull request closed without merging**